lib/oidc/mixin/index.ts

```diff
@@ -86,19 +85,7 @@ export function mixinOAuth
       // TokenManager
       this.tokenManager = new TokenManager(this, this.options.tokenManager);
 
-      const _setLocation = (url) => {
-        if (this.options.setLocation) {
-          this.options.setLocation(url);
-        } else {
-          window.location = url;
-        }
-      };
-      const enrollAuthenticatorFn = enrollAuthenticator.bind(null, this) as EnrollAuthenticatorFunction;
-      const enrollAuthenticatorApi: EnrollAuthenticatorAPI = Object.assign(enrollAuthenticatorFn, {
-        // This is exposed so we can set window.location in our tests
-        _setLocation
-      });
-      this.enrollAuthenticator = enrollAuthenticatorApi;
+      this.enrollAuthenticator = enrollAuthenticator.bind(null, this) as EnrollAuthenticatorFunction;
```

Comment:
prefer to avoid this explicit type-cast. proposed refactor: https://github.com/okta/okta-auth-js/compare/od-mfa_enroll-OKTA-539548...ag-mfa_enroll-OKTA-539548?expand=1